src/header.js

```diff
@@ -5,53 +5,35 @@ class Header extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      searchActive: "",
+      searchActive: false,
       searchText: "",
     };
 
-    this.handleDiggClick = this.handleDiggClick.bind(this);
-    this.handleRedditClick = this.handleRedditClick.bind(this);
-    this.handleMashableClick = this.handleMashableClick.bind(this);
-    this.handleFeedrClick = this.handleFeedrClick.bind(this);
+    this.selectFeed = this.selectFeed.bind(this);
     this.handleSearchClick = this.handleSearchClick.bind(this);
     this.handleSearchChange = this.handleSearchChange.bind(this);
   }
 
-  handleSelectSource() {
```

Comment:
I noticed that you weren't using this function anymore.
I always recommend cleaning up unused code as you go - but it looks like you might have just missed this one.